### PR TITLE
[GUI] [Trivial] Rebranding work

### PR DESCRIPTION
**Pull request by JSKitty**
This PR rebrands the vast majority of the ZENZO Core v2.0 codebase, user-viewable elements such as the QT and CLI wallets, as well as the codebase itself, methods, comments and variables for consistency.

This PR should NOT change the functionality of ZENZO Core, but simply rebrand and rename the codebase.

---
This should be fairly easy to review, even for non-devs, as there isn't "code" added/removed, it's simply "PIVX --> ZENZO", "PIV --> ZNZ", etc. If anything looks inconsistent, out-of-place, or something was missed; Please leave a review!

### CODE REVIEW

File: src/init.cpp
Comment (YuurinBee) [inline src/init.cpp:1765]: Should this be changed to zZNZWallet?

File: src/init.cpp
Comment (YuurinBee) [inline src/init.cpp:1769]: Should zpivTracker be zZNZTracker?

File: src/qt/pivx/settings/forms/settingsfaqwidget.ui
Comment (YuurinBee) [inline src/qt/pivx/settings/forms/settingsfaqwidget.ui:1242]: "There package will be lit up"? 

Grammar Suggestions: There, the package will be lit up

Proper Suggestion: There, replace package icon with staking icon and for the second sentence... 

There, the staking icon will be glowing white and will state &quot;Staking Enabled&quot; to indicate it is staking.

File: src/masternode-budget.h
Comment (YuurinBee) [inline src/masternode-budget.h:368]: I believe this should be:

Total ZNZ paid out by this budget

**Review by YuurinBee — CHANGES_REQUESTED** [review]
Commented with suggestions, awaiting @JSKitty 's feedback/approval before final approval + merge.

File: src/init.cpp
Comment (JSKitty) [reply src/init.cpp:1765]: Yup

File: src/init.cpp
Comment (JSKitty) [reply src/init.cpp:1769]: Damn, not sure how I missed this, could lead to compiling errors, will resolve.

File: src/qt/pivx/settings/forms/settingsfaqwidget.ui
Comment (JSKitty) [reply src/qt/pivx/settings/forms/settingsfaqwidget.ui:1242]: Could be amended in a seperate PR.

File: src/masternode-budget.h
Comment (JSKitty) [reply src/masternode-budget.h:368]: That works, non-blocking nit though, possibly for another PR.

File: src/init.cpp
Comment (JSKitty) [reply src/init.cpp:1765]: Actually, just re-ran the compiler and it's all fine (I did this yesterday, but double-checking).
Since this is Zerocoin code (And there's a bunch of PIV branded zerocoin code still), it may be better to just leave it, as we will be removing it all anyway in another PR, 99% of our Zerocoin code will be purged soon.

**Comment by JSKitty** [comment]
PR compiles well and the majority of the needed codebase is rebranded, any PIVX-branded Zerocoin code will be removed entirely in the near future, and minor nits will be left for an additional PR.

**Pull request merged**
